doc/source/performance/innodb_numa_support.rst

```diff
@@ -14,6 +14,11 @@ Version Specific Information
  * :rn:`5.7.10-1`: 
     Feature ported from |Percona Server| 5.6
 
+ * :rn:`5.7.22-22`:
+    Feature reverted from the upstream implementation back to the original one,
```

Comment:
5.7 never had the better-than-upstream feature, so this should be something like "such and such improvements ported from 5.6" or similar

Reply:
Rewritten